src/connectionhealthpolicy/SendingAudioFailureConnectionHealthPolicy.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../logger/Logger';
+import BaseConnectionHealthPolicy from './BaseConnectionHealthPolicy';
+import ConnectionHealthData from './ConnectionHealthData';
+import ConnectionHealthPolicy from './ConnectionHealthPolicy';
+import ConnectionHealthPolicyConfiguration from './ConnectionHealthPolicyConfiguration';
+
+export default class SendingAudioFailureConnectionHealthPolicy
+  extends BaseConnectionHealthPolicy
+  implements ConnectionHealthPolicy {
+  private readonly secondsToConsider: number;
+  private readonly initialWaitTimeMs: number;
+  private readonly coolDownTimeMs: number;
+  private readonly maximumTimesToWarn: number;
+  private warnCount: number;
+  private lastWarnTimestampMs: number;
+
+  constructor(
+    private logger: Logger,
+    configuration: ConnectionHealthPolicyConfiguration,
+    data: ConnectionHealthData
+  ) {
+    super(configuration, data, 'Sending Audio Health');
+    this.secondsToConsider = configuration.sendingAudioFailureSamplesToConsider;
```

Comment:
[For understanding]: Failure samples here means audio packets metric or how? Though private, the `secondsToConsider` might need a better naming 😅 .

Reply:
Sure, that makes sense!